cli/src/semgrep/dependency_aware_rule.py

```diff
@@ -100,7 +100,7 @@ def generate_unreachable_sca_findings(
                         path=out.Fpath(str(lockfile_path)),
                         start=out.Position(found_dep.line_number or 0, 0, 0),
                         end=out.Position(
-                            (found_dep.line_number + 1 if found_dep.line_number else 0),
+                            (found_dep.line_number if found_dep.line_number else 0),
```

Comment:
The main change of this PR

Reply:
as we discussed, this *does* break highlighting for these findings. One line will be displayed but it will not be highlighted. I don't think there will be confusion around meaning, but it will be aesthetically inconsistent with other findings

Reply:
Yeah. I think removal of extra lines make it fine.